packages/components/src/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { useTabList } from '@react-aria/tabs';
+import {
+  Box,
+  ThemeExtensionsWithParts,
+  useComponentStyles,
+} from '@marigold/system';
+import { useTabListState } from '@react-stately/tabs';
+import { useRef } from 'react';
+import { AriaTabListProps } from '@react-types/tabs';
+
+import { Tab } from './Tab';
+import { TabPanel } from './TabPanel';
+import { TabContext } from './Context';
+import { Item } from '@react-stately/collections';
+
+export interface TabsThemeExtension
+  extends ThemeExtensionsWithParts<'Tabs', ['tabs', 'tab', 'tabPanel']> {}
+export interface TabsProps
+  extends Omit<AriaTabListProps<object>, 'orientation'> {
+  gap?: number;
```

Comment:
This doesn't need to be expoed here, a `variant` in the `theme` can override this.